studio/lib/constants/prism.ts

```diff
@@ -29,7 +29,7 @@ export function dart(Prism: any) {
       className,
       {
         // variables and parameters
-        // this to support class names (or generic parameters) which do not contain a lower case letter (also works for methods)
+        // add this to support class names (or generic parameters) which do not contain a lower case letter (also works for methods)
```

Comment:
Good PR 🎯 just a small suggestion for clarity - could you update the phrasing from "add this to support" to "supports class names"? It reads a bit smoother and is more concise.

Reply:
sure . done !!